.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.20-openshift-4.15
+  tag: rhel-9-release-golang-1.21-openshift-4.16
```

Comment:
This commit could be also squashed into `UPSTREAM: <carry>: Add OpenShift files`